adapters/adnuntius/adnuntius.go

```diff
@@ -102,47 +105,47 @@ func makeEndpointUrl(ortbRequest openrtb2.BidRequest, endpoint string) (string,
 	q.Set("tzo", fmt.Sprint(tzo))
 	q.Set("format", "json")
 
-	url := endpoint + "?" + q.Encode()
+	url := a.endpoint + "?" + q.Encode()
 	return url, nil
 }
 
 /*
 	Generate the requests to Adnuntius to reduce the amount of requests going out.
 */
-func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters.RequestData, error) {
+func (a *adapter) generateRequests(ortbRequest openrtb2.BidRequest) ([]*adapters.RequestData, []error) {
 	var requestData []*adapters.RequestData
+	var errors []error
 	networkAdunitMap := make(map[string][]adnAdunit)
 	headers := setHeaders()
-	endpoint, err := makeEndpointUrl(ortbRequest, a.endpoint)
+
+	endpoint, err := makeEndpointUrl(ortbRequest, a)
 	if err != nil {
-		return nil, fmt.Errorf("failed to parse URL: %v", err)
+		errors = append(errors, fmt.Errorf("failed to parse URL: %v", err))
 	}
 
 	for _, imp := range ortbRequest.Imp {
 		if imp.Banner == nil {
-			return nil, &errortypes.BadInput{
+			errors = append(errors, &errortypes.BadInput{
 				Message: fmt.Sprintf("ignoring imp id=%s, Adnuntius supports only Banner", imp.ID),
-			}
+			})
 		}
 		var bidderExt adapters.ExtImpBidder
 		if err := json.Unmarshal(imp.Ext, &bidderExt); err != nil {
-			return nil, &errortypes.BadInput{
+			errors = append(errors, &errortypes.BadInput{
 				Message: fmt.Sprintf("Error unmarshalling ExtImpBidder: %s", err.Error()),
-			}
+			})
 		}
 
 		var adnuntiusExt openrtb_ext.ImpExtAdnunitus
 		if err := json.Unmarshal(bidderExt.Bidder, &adnuntiusExt); err != nil {
-			return nil, &errortypes.BadInput{
+			errors = append(errors, &errortypes.BadInput{
 				Message: fmt.Sprintf("Error unmarshalling ExtImpBmtm: %s", err.Error()),
-			}
+			})
 		}
 
-		var network string
+		network := "default"
```

Comment:
This is because it might change in the lines below, so making it a constant is not correct, I think? :) but then again my first go at golang, (pun intended) so i might be mistaking.